wiki/Tournaments/TEMPLATE.md

```diff
@@ -0,0 +1,148 @@
+---
+tags:
+  - {SHORT_TITLE}
+  - {SERIES_SHORT_TITLE}
+---
+
+# {TITLE}
+
+![{SHORT_TITLE} logo/banner](img/logo.png) <!-- choose the option that suits the image better -->
+
+The **{TITLE}** (***{SHORT_TITLE}***) is a <!-- enter applicable tournament traits, example: {N}v{N}, double-elimination, country-based --> {MODE} tournament hosted by {ORGANISERS} <!-- list the organisers here with their flags -->. It is the {ITERATION_ORDINAL} instalment of the {SERIES_TITLE}.
```

Comment:
Hadn't quite thought about/noticed this before, but not all tournament titles can really take a "the".

For example, `the Corgi Cup` and `the osu! World Cup` work, while `the Corgi Cup V` or `the osu! World Cup 2020` do not because of the number. `thing #20` works but `the thing #20` implies that `the thing` is a thing of its own (i.e. "The Thing"). Something like `the {TITLE} tournament` would always work, albeit clumsily.

Not entirely sure what's the best way to convey this in the template. Perhaps remove the first "the" (more likely to be correct that way) and add a note somewhere explaining to add or remove it when it makes sense.

```suggestion
**{TITLE}** (***{SHORT_TITLE}***) is a <!-- enter applicable tournament traits, example: {N}v{N}, double-elimination, country-based --> {MODE} tournament hosted by {ORGANISERS} <!-- list the organisers here with their flags -->. It is the {ITERATION_ORDINAL} instalment of the {SERIES_TITLE}.
```

Reply:
really not sure about that, i personally would leave it how it its right now 

checking Wikipedia, they do the same for tournament articles:

![image](https://user-images.githubusercontent.com/19843905/130372556-bd4ea99b-962d-4a59-90b1-8be17f15b900.png)

![image](https://user-images.githubusercontent.com/19843905/130372561-623e2cd9-f7e4-4608-a567-b15d9095c110.png)

Reply:
The problem I have with it at the moment is that, for some reason, osu! seems to like having tournaments named with possessives, so tournaments like `Yaz's Summer Cup`, `Nik's Summer Tour` and `Dio's Bizarre Fall Classic` all just read kinda clunkily when prefixed with `the` i.e. `The Yaz's Summer Cup 2021` etc.

We could mostly resolve this by moving the year to the front of the title in the paragraph text so `The Yaz's Summer Cup 2021` would become `The 2021 Yaz's Summer Cup`. Wikipedia seems to do it for some articles and should work in some cases for us too:

![image](https://user-images.githubusercontent.com/9306592/130373656-06c2365f-faa2-4c7f-a0bf-919bf864754b.png)

_The ICC who organizes the event calls it the `ICC Cricket World Cup 2019`, but Wikipedia refers to it differently._
![image](https://user-images.githubusercontent.com/9306592/130373783-f6c4d85e-b339-45c7-b181-5247ae8f2770.png)

The main downside that I can see is that numbered iterations (`#5`) wouldn't work at the front, so it would only apply to tournaments that iterate using years.

Reply:
well even if we do go by that convention, some tournament names would be wrong no matter how you put it, namely:

- [112's Tournaments](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/112/en.md)
- [nik's Tournaments](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/NT/en.md)
- [Yaz's Summer Cup](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/YSC/en.md)
- [Taiko Last Man Standing](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/T_LMS/en.md) (esp. "Taiko: Last Man Standing")
- [Catch French Beginner](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/CFB/en.md)*
- [osu! Cup Master](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/OCM/en.md)*
- [Speed of Light](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/SOL/en.md)* (only for the series' name)
- [Most Amazing LAN Ever](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/osu!_Riverside/MALE/en.md)*

\* works, but would have to pretend that "The" is part of the title, which it is not

for all of these, you could also just append "series" to the series title and "tournament" after the tournament title if it's not already part of it, but i'd just prefer not having a "the" for them

Reply:
Couldn't you just put "the" in brackets here to side-step the issue? Making sure every single article starts off with the word "the" isn't really worth the trouble if it means making titles sound like your reading a tongue twister imo

Reply:
this is a starter template, not a precise grammar... if it usually works, keep it, and if not, delete it

Reply:
i'm fine with keeping the template as-is, as it does have the disclaimer on grammar around placeholders, but some consensus on the "the" matter here would help in knowing whether any so-considered exceptions should be changed in the articles or if we'll just always have it one way